design/ceph/ceph-nfs-ganesha.md

```diff
@@ -129,6 +130,13 @@ spec:
             configMap:
               name: rook-ceph-nfs-organization-sssd-config
               defaultMode: 0600 # mode must be 0600
+        sssdGenericFiles:
+          volumeSource:
+            # example
+            configMap:
+              name: rook-ceph-nfs-organization-sssd-ca-bundle
+              defaultMode: 0600 # mode must be 0600
```

Comment:
If we are including supplemental files, it might be better to allow users to specify as many mounts as they like. We could make `volumeSource` a list

```suggestion
          - volumeSource:
              configMap:
                name: rook-ceph-nfs-organization-sssd-ca-bundle
                defaultMode: 0600 # mode must be 0600
          - volumeSource:
              hostPath:
                # etc.
```